etc/rita.yaml

```diff
@@ -56,9 +56,44 @@ Bro:
     # records unimported if there is an error
     ImportBuffer: 100000
 
-SafeBrowsing:
-    APIKey: ""
-    Database: $HOME/.rita/safebrowsing
+BlackListed:
+    Database: rita-blacklist
+    # These are blacklists built into rita-blacklist. Set these to false
+    # to disable checks against them.
+    myIP.ms: true
+    MalwareDomains.com: true
+    MalwareDomainList.com: true
+
+    # Google SafeBrowsing requires an api key and a file to cache the results in.
+    # If either APIKey or Database is an empty string, Google SafeBrowsing will
+    # not be queried.
+    SafeBrowsing:
+        APIKey: ""
+        Database: $HOME/.rita/safebrowsing
+
+    # These are custom blacklists that you may define. They are lists of either
+    # file paths or urls. These custom blacklists are expected to be simple,
+    # line separated text documents containing a list of blacklisted entries.
+
+    # Example: CustomIPBlacklists: ["$HOME/.rita/myIPBlacklist.txt"]
+    # myIPBlacklist.txt would look like this:
+    # 192.168.0.1
+    # 10.10.174.1
+
+    # Lists containing both IPv4 and IPv6 addresses are acceptable
+    CustomIPBlacklists: []
+    # Lists containing hostnames, domain names, and FQDNs are acceptable
+    CustomHostnameBlacklists: []
+    # URLs must each contain a protocol, a host, and a resource
+    # Ex: http://google.com/
+    # Ex: ftp://myftpserver.com/a/file/over/here.txt
```

Comment:
On [this line](https://github.com/ocmdev/rita/pull/100/files#diff-31394ce29daf4599be7f6fb16b6516a4R100), it looks like the url prefix is hardcoded to http...  Will other prefixes still work and if so, how?